tools/run-layout.sh

```diff
@@ -16,7 +16,7 @@
 
 mkdir -p .test
 cd .test
-[ -d zuul ] || git clone https://git.openstack.org/openstack-infra/zuul --depth 1
+[ -d zuul ] || git clone https://git.openstack.org/openstack-infra/zuul --depth 1 -b 2.6.0
```

Comment:
Var'ify this ver #, comment on why we chose it.

Reply:
> Var'ify this ver #, comment on why we chose it.
We chose 2.6.0 as it was the last version to support jenkins, zuul 3.x was an entire rewrite.